### ISSUE TITLE
Another strange crash

### ISSUE THREAD

**Issue by blackborn66** [gooofy/aqb#36]
The following lines let the IDE crash
I tested in 0.8.2 and 0.9.0

```
OPTION EXPLICIT

DIM SHARED colr AS Integer

SUB test (split AS Integer)
    colr = split + 4 REM this works
    colr = 16 - split REM this leads to crash
END SUB
```

**Comment by gooofy** [comment]
again, this one works for me - I suspect my earlier fixes for #42 and #44 may have fixed this one as well

**Comment by blackborn66** [comment]
okay
